docs/topics/ref_resolved-issues-7-1-1.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.1/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="resolved-issues-7-1-1_{context}"]
+= Resolved issues
+
+This section provides highlighted issues that have been resolved in {ProductFullName} 7.1.1.
+
+.Application analysis no longer fails when run with the `--bulk` option
+
+Previously, if you ran an application analysis with the `--bulk` option, it failed with the following error:
+----
+ERRO[0143] container run error                       
+ERRO[0143] failed to generate static report
+----
+
+With this update, the issue has been resolved, and the bulk analysis completes without errors and generates a proper analysis report. Note, however, that the bulk analysis still fails on Windows systems. (link:https:https://issues.redhat.com/browse/MTA-3663[MTA-3663])
+
+.Enabling Preemption now works as expected if authentication is enabled
+
+Previously, *Preemption* could not be enabled for tasks in the *Task Manager* page and the Task Manager drawer if {ProductFullName} authentication was enabled. With this update, the issue has been resolved, and you can now enable *Preemption* if {ProductShortName} authentication is enabled.(link:https:https://issues.redhat.com/browse/MTA-3195[MTA-3195])
+
+.The drop-down application menu no longer contains duplicate options
+
+Previously, when selecting a target for application analysis, the drop-down menu contained duplicate application target options. With this update, the issue has been resolved, and no duplicates are listed when selecting the application analysis target. (link:https:https://issues.redhat.com/browse/MTA-2795[MTA-2795])
+
+.The Reports page now displays separate charts for unassessed applications and applications with the Unknown risk
+
+Previously, the {ProductFullName} user interface (UI) did not distinguish between unassessed applications and applications that have been assessed to have the `Unknown` risk. Consequently, the *Reports* page displayed a single chart for such applications. With this update, the issue has been resolved, and the *Reports* page now displays separate charts for unassessed applications and applications with the *Unknown* risk. (link:https:https://issues.redhat.com/browse/MTA-2678[MTA-2678])
+
+.Analyzing public applications no longer fails
```

Comment:
This should say "Fixed issue where some analysis with custom rules were erroring"

Reply:
Cool will make the change but i refuse to use the word `erroring`... so pedantic LOL

Reply:
@dymurray, do I get it correctly that like only certain rules caused some errors which would could analysis failure? If it is like that, I think the whole release note description should be rewritten....

Reply:
@mpershina && @vsodhi - updated and @raoradhika is going to merge shortly